apps/i18n/common/en_us.json

```diff
@@ -370,16 +370,12 @@
   "deleteAccountDialog_body3": "It will also **permanently delete your students’ accounts** unless they created a personal login.",
   "deleteAccountDialog_button": "Delete my Account",
   "deleteAccountDialog_button_studentWarning": "Delete my and my students' accounts",
-  "deleteAccountDialog_checkboxPreface": "I understand that by deleting my account, ",
   "deleteAccountDialog_checkboxTitle": "Please verify the following {numCheckboxes, plural, one {statement} other {statements}} before you can delete your account:",
-  "deleteAccountDialog_checkbox1_1": "I have the authority to delete the education records ",
-  "deleteAccountDialog_checkbox1_2": "of the students in my sections (or release it to them, if they have a personal login). ",
-  "deleteAccountDialog_checkbox2_1": "I am aware of the ",
-  "deleteAccountDialog_checkbox2_2": "message to send to parents ",
-  "deleteAccountDialog_checkbox2_3": "to warn them that their children’s coding projects will be deleted unless they act to preserve it by creating a personal login for their children.",
-  "deleteAccountDialog_checkbox3": "my students’ accounts may also be permanently deleted.",
-  "deleteAccountDialog_checkbox4": "my students may not be able to access their accounts anymore.",
-  "deleteAccountDialog_checkbox5": "the projects and creations of my students may also be deleted.",
+  "deleteAccountDialog_checkbox1": "**I have the authority to delete the education records** of the students in my sections (or release it to them, if they have a personal login).",
+  "deleteAccountDialog_checkbox2": "I am aware of the [message to send to parents](https://support.code.org/hc/en-us/articles/115001475131-Adding-a-personal-login-to-a-teacher-created-account) to warn them that their children’s coding projects will be deleted unless they act to preserve it by creating a personal login for their children. ",
```

Comment:
I'm not sure how I feel about hardcoding the URL into the string like this. On the one hand, we are moving toward a system that will allow for this to be done safely, without the possibility for translator manipulation. On the other hand, we're not there yet and I'm not sure when we will be.

I think for now, I'd suggest building this with a template variable like

    "deleteAccountDialog_checkbox2": "I am aware of the [message to send to parents]({explanationUrl}) to warn them that their children’s coding projects will be deleted unless they act to preserve it by creating a personal login for their children. ",

Then using it in the component like

    <UnsafeRenderedMarkdown
      markdown={i18n.deleteAccountDialog_checkbox1({
        explanationUrl: RELEASE_OR_DELETE_RECORDS_EXPLANATION
      )}
    />